### ISSUE TITLE
Package is published with .idea IDE configuration files

### ISSUE THREAD

**Issue by thibaudcolas** [shakacode/sass-resources-loader#126]
The published package contains your IDE’s configuration files, which I assume is unintended:

```
$ la node_modules/sass-resources-loader
total 64
drwxr-xr-x  11 thibaudcolas  staff   352 Aug 26 15:44 .idea/
-rw-r--r--   1 thibaudcolas  staff  2004 Oct 26  1985 CHANGELOG.md
-rw-r--r--   1 thibaudcolas  staff  1435 Oct 26  1985 CODE_OF_CONDUCT.md
-rw-r--r--   1 thibaudcolas  staff  2582 Oct 26  1985 CONTRIBUTING.md
-rw-r--r--   1 thibaudcolas  staff  1053 Oct 26  1985 LICENSE
-rw-r--r--   1 thibaudcolas  staff  7136 Oct 26  1985 README.md
-rw-r--r--   1 thibaudcolas  staff   115 Oct 26  1985 jest.config.js
drwxr-xr-x   4 thibaudcolas  staff   128 Aug 26 15:44 lib/
-rw-r--r--   1 thibaudcolas  staff  2618 Aug 26 15:45 package.json
```

I’d suggest removing the `.npmignore` file, and using the package.json’s `files` field instead:

```json
"files": [
    "lib"
  ],
```

This will result in the list below (a few files are included no matter what):

```
$ la node_modules/sass-resources-loader
total 64
-rw-r--r--   1 thibaudcolas  staff  2004 Oct 26  1985 CHANGELOG.md
-rw-r--r--   1 thibaudcolas  staff  1053 Oct 26  1985 LICENSE
-rw-r--r--   1 thibaudcolas  staff  7136 Oct 26  1985 README.md
drwxr-xr-x   4 thibaudcolas  staff   128 Aug 26 15:44 lib/
-rw-r--r--   1 thibaudcolas  staff  2618 Aug 26 15:45 package.json
```

**Comment by justin808** [comment]
I agree. @thibaudcolas could you create a PR and confirm this works for you?

**Comment by justin808** [comment]
Fixed in #128
